### ISSUE TITLE
Automatic change of page background color for discontinued projects? 

### ISSUE THREAD

**Issue by LeeteqXV** [backdrop-ops/backdropcms.org#410]
Coming from: https://github.com/backdrop-contrib/base_meta/issues/2 and https://github.com/backdrop-contrib/seo_meta/issues/8

**When a project is cancelled, maybe we should have (exists?) a category for that which also automagically colored the background of the project page somewhat differently (light redish?) to notify any page visitor of that status.**

**Comment by jenlampton** [comment]
Yes, that's a good idea. Do you think the `exists` flag should be some kind of a manual field on backdropcms.org, or something that's pulled in automatically from GitHub?

**Comment by jenlampton** [comment]
Maybe this is something we should add into the info file, much like dependancies. If a module is being discontinued in favor of another, we should link to the new project too. so something like...

```
replaced_by[] = seo_meta
```

This could cause a notice on the backdrop site as well as display something special on backdropcms.org.
